.circleci/config.yml

```diff
@@ -76,6 +79,7 @@ jobs:
 
   npm_release:
     <<: *node_executor
+    resource_class: medium+
```

Comment:
Just curious, why `sdk` needs this?

Reply:
Seems like there are too many deprecated packages, so while running npm ci memory in circle machine is maxed out and the process gets killed. We will need to come back and fix the packages later.